var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -403,14 +406,17 @@ def install(self, spec, prefix):
             if spec.satisfies('^trilinos+boost'):
                 options.append('--with-boost=1')
 
-        if self.spec.satisfies('clanguage=C++'):
+        if spec.satisfies('clanguage=C++'):
             options.append('--with-clanguage=C++')
         else:
             options.append('--with-clanguage=C')
 
         # Activates library support if needed (i.e. direct dependency)
         jpeg_sp = spec['jpeg'].name if 'jpeg' in spec else 'jpeg'
-        scalapack_sp = spec['scalapack'].name if 'scalapack' in spec else 'scalapack'
+        if 'scalapack' in spec:
+            scalapack_sp = spec['scalapack'].name
+        else:
+            scalapack_sp = 'scalapack'
```

Comment:
I suspect this change can also be reverted

Reply:
Yes, it just sneaks inside the flake8 line length!